src/app.ts

```diff
@@ -77,41 +81,70 @@ app.get("/map", (req, res) => {
   } as MapData);
 });
 
-app.get("/range-timeseries-data", (req, res) => {
-  let keys: string[] = Object.keys(db);
-  const dataForAllInterventions = keys.map((key) => {
-    let dataToProcess = db[key];
-    const rowsGroupedByDate: Map<number, Entry[]> = new Map<number, Entry[]>();
-    const stateCode: string = req.query.stateCode;
-    if (stateCode != null) {
-      dataToProcess = dataToProcess.filter((row) =>
-        row.county.endsWith(stateCode)
-      );
-    }
+app.get("/timeseries-data", (req, res) => {
+  const contacts = Object.keys(db);
+  const stateCode = req.query.stateCode;
+  const type: string = req.query.type;
+  const percentiles = ["2.5", "25", "50", "75", "97.5"];
```

Comment:
I think we can extract these from the DB instead of hardcode them here.

Reply:
If it's difficult to change that you can ignore this comment. I guess it's unlikely for them to add more percentiles so hardcoding in the backend should be fine. But let's make sure the front-end doesn't need hardcoding.

Reply:
yup ! if it turns out they add more, i will remove the hardcoding.